src/eth/command/network/create.ts

```diff
@@ -37,11 +47,11 @@ export const handler = async (argv: Arguments<OptType>) => {
       const confirmDelete = (await prompts({
         type: 'confirm',
         name: 'value',
-        message: '⚠️ Detecting quorum nodes already exists. The following processes will remove all existing files. Continue?',
+        message: `⚠️ Detecting ${networkType} nodes already exists. The following processes will remove all existing files. Continue?`,
```

Comment:
if I create a network Besu before, but new generate one choose quroum. Wrong output

Reply:
should use fs to read files setting, not networktype

Reply:
Fixed this issue and delete origin network when create a new one.